forge-gui/res/cardsfolder/a/akim_the_soaring_wind.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:2 U R W
 Types:Legendary Creature Bird Dinosaur
 PT:3/4
 K:Flying
-T:Mode$ TokenCreated | ValidPlayer$ You | OnlyFirst$ True | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you create one or more tokens for the first time each turn, create a 1/1 white Bird creature token with flying.
+T:Mode$ TokenCreated | ValidToken$ Card.YouOwn | OnlyFirst$ True | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever you create one or more tokens for the first time each turn, create a 1/1 white Bird creature token with flying.
```

Comment:
imo this should be a TokenCreatedOnce trigger

Reply:
The thing is TokenCreatedOnce might contain tokens with different owners
So we'd need to build a `Map<Player, Boolean> firstTimers` and then check that in the trigger

This feels like unnecessary overhead?

Reply:
Or: make a List of Players that have their token created for the first time imo List would be better than map?
and then use `FirstTime$ You`